mettle/src/stdapi/webcam/webcam.c

```diff
@@ -16,7 +16,7 @@
 #include "log.h"
 #include "tlv.h"
 
-#if __APPLE__
+#if __APPLE__ || __linux__
```

Comment:
I might suggest rather than adding OS defines in the code, add #ifdef HAVE_WEBCAM and then add a define that gets set in configure.ac